torch/_inductor/codegen/cuda/gemm_template.py

```diff
@@ -1374,8 +1374,6 @@ def _are_inputs_layout_compatible(self, layouts: List[Layout]) -> bool:
         A_size = [int(i) for i in A_layout.size]
         B_size = [int(i) for i in B_layout.size]
         K = max(A_size[-1], B_size[-2])
-        M = A_size[-2]
-        N = B_size[-1]
```

Comment:
tbh I think it would be best for readability to change calls to these sizes below with M and N rather than removing them.